src/AppInstallerCLIE2ETests/SourceCommand.cs

```diff
@@ -7,9 +7,8 @@ namespace AppInstallerCLIE2ETests
 
     public class SourceCommand
     {
-        // Todo: switch to use prod index when available
-        private const string SourceTestSourceUrl = @"https://winget-int.azureedge.net/cache";
         private const string SourceTestSourceName = @"SourceTestSource";
+        private const string SourceTestSourceUrl = @"https://localhost:5001/TestKit";

```

Comment:
>SourceTestSourceUrl [](start = 29, length = 19)

I'm surprised that this works; I would expect it to fail due to adding the same URL twice.

But to fix the fact that this test class will trash the source state, the source setup should actually happen in a base class that all of the test classes derive from.  That way, any individual test class can't break the following ones:
```
class Base
{
  [OneTimeSetup]
  Setup()
  { // set up the source list to only be the test source [and I would start with a force reset to ensure it] }

  [OneTimeTeardown]
  Teardown()
  { // source reset }
}

...

class SourceCommand : Base
{
}
```